stdlib/public/runtime/Exclusivity.cpp

```diff
@@ -171,8 +182,18 @@ class AccessSet {
   Access *Head = nullptr;
 public:
   constexpr AccessSet() {}
+  constexpr AccessSet(Access *Head) : Head(Head) {}
+
+  constexpr operator bool() const { return bool(Head); }
+  constexpr Access *getHead() const { return Head; }
+  void setHead(Access *newHead) { Head = newHead; }
+  constexpr bool isHead(Access *access) const { return Head == access; }
 
   bool insert(Access *access, void *pc, void *pointer, ExclusivityFlags flags) {
+#ifndef NDEBUG
+    if (isExclusivityLoggingEnabled())
+      printf("Inserting new access: %p\n", access);
```

Comment:
Outputting these to stderr would probably be a little better.

Reply:
You are right. I was doing it to help me writing the tests. But I was being lazy. I'll fix it.